test/integration/groups_test.go

```diff
@@ -166,6 +166,10 @@ var _ = Describe("Groups", func() {
 			Expect(group1.Clusters[0].ClusterID).To(Equal(newClusterDetails.ClusterID))
 			Expect(group1.Clusters[0].Name).To(Equal(clusterName))
 
+			//Remove cluster from group
+			_, err = c.Groups.UnGroupClusters(testConfig.OrgID, newGroupDetails.UUID, []string{newClusterDetails.ClusterID})
+			Expect(err).NotTo(HaveOccurred())
+
```

Comment:
Just testing that it doesn't error out isn't enough. I suggest the following (at each step `Expect`ing there is no error, of course):
- Create (register) another couple of clusters
- Add those to the group
- Use your new method to remove one of the clusters
- List the clusters in the group
- Verify that the cluster you just removed is no longer there, but the other clusters still are

Reply:
Added Expects to make sure cluster was getting removed from group